pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -457,12 +481,16 @@ func (qjm *XController) PreemptQueueJobs() {
 		if err := qjm.updateEtcd(updateNewJob, "PreemptQueueJobs - CanRun: false"); err != nil {
 			klog.Errorf("Failed to update status of AppWrapper %v/%v: %v", aw.Namespace, aw.Name, err)
 		}
-		klog.V(4).Infof("[PreemptQueueJobs] Adding preempted AppWrapper %s/%s to backoff queue.",
-			aw.Name, aw.Namespace)
-		go qjm.backoff(aw, "PreemptionTriggered", string(message))
-
+		if cleanAppWrapper {
+			klog.V(4).Infof("[PreemptQueueJobs] Deleting AppWrapper %s/%s due to maximum number of requeuings exceeded.", aw.Name, aw.Namespace)
+			go qjm.Cleanup(aw)
```

Comment:
@metalcycling do we know what is clean up deleting? appwrapper or child objects owned by appwrapper?

Reply:
It cleans up all the `GenericItems` The AppWrapper is left alone and it can be inspected with `oc get appwrappers`. It is just removed for the dispatching queues so it is never retried again.

Reply:
ok great!